speechbrain/k2_integration/losses.py

```diff
@@ -0,0 +1,152 @@
+""" This file contains the loss functions for k2 training. Currently, we only
+support CTC loss.
+
+Authors:
+ * Zeyu Zhao 2023
+ * Georgios Karakasidis 2023
+"""
+try:
+    import k2
+except ImportError:
+    MSG = "Please install k2 to use k2 training \n"
+    MSG += "E.G. run: pip install k2\n"
+    raise ImportError(MSG)
+
+import torch
+
+
+def ctc_k2(
```

Comment:
We need an example for this function so that it is tested and also easier to use by our users.